### PR TITLE
docs: update SECURITY.md

### PR DESCRIPTION
**Pull request by lehnberg**
as discussed with protocol devs:
- add explicit security policy
- simplify bug bounty program
- reference security team and /yearn-security
- add myself as point of contact

### CODE REVIEW

**Comment by lehnberg** [comment]
Blocked by: https://github.com/iearn-finance/yearn-security/pull/4

File: SECURITY.md [hunk 1]
```diff
@@ -1,84 +1,93 @@
...
+For all security related issues, Yearn has 3 main points of contact:
 
-**Likely** - exploitable by anyone, under certain conditions (e.g., user with x amount of funds locked, or exploitable only over a longer period of time).
+| Contact                | Public key                                                                                                   | Email                             | Keybase                                         |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------ | --------------------------------- | ----------------------------------------------- |
+| milkyklim              | [PGP](https://github.com/iearn-finance/yearn-security/blob/master/keys/milkyklim.asc)                        | y at milkyklim.com                | [@milkyklim](https://keybase.io/milkyklim/chat) |
+| Doggie Boy / fubuloubu | [PGP](https://github.com/iearn-finance/yearn-security/blob/master/keys/publickey.yearn-security%40pm.me.asc) | yearn-security at pm.me           | N/A                                             |
```
Comment (milkyklim) [inline SECURITY.md:35]: @fubuloubu Doggie, you have to change the path to your PGP which sits in another repo. See https://github.com/iearn-finance/yearn-security/issues/6

**Comment by lehnberg** [comment]
Thanks @milkyklim, agree with the fixes, looks good 👍

File: SECURITY.md (hunk 1 above)
Comment (milkyklim) [reply SECURITY.md:35]: As discussed, Doggie will add his PGP later.

**Review by milkyklim — APPROVED** [review]

**Pull request merged**